permissions/migrations/0029_migrate_permissions_to_new_models.py

```diff
@@ -88,14 +94,16 @@ def migrate_permissions_to_new_models(apps, schema_editor):
     NewUnitRoleUnitGroup.objects.bulk_create(new_unit_roles_unit_groups)
 
 
-def _determine_role(permissions: list[str]) -> str:
-    if permissions == RESERVER_PERMISSIONS:
+def _determine_role(permissions: list[str], role_name: str) -> str:
+    # Role names can technically be anything, but these names are used in production,
+    # which is the most critical part for the migration.
+    if role_name in ["Varaaja", "viewer"] or permissions == RESERVER_PERMISSIONS:
```

Comment:
`"viewer"` to `UserRoleChoices.VIEWER` with `"Katselija"`

Reply:
Whoops. FIxed.